sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -516,6 +505,35 @@ case class DescribeTableCommand(
       new MetadataBuilder().putString("comment", "comment of the column").build())()
   )
 
+  protected def describeSchema(
+      schema: StructType,
+      buffer: ArrayBuffer[Row],
+      header: Boolean): Unit = {
+    if (header) {
+      append(buffer, s"# ${output.head.name}", output(1).name, output(2).name)
+    }
+    schema.foreach { column =>
+      append(buffer, column.name, column.dataType.simpleString, column.getComment().orNull)
+    }
+  }
+
+  protected def append(
+    buffer: ArrayBuffer[Row], column: String, dataType: String, comment: String): Unit = {
+    buffer += Row(column, dataType, comment)
+  }
+}
+/**
+ * Command can of the following form.
```

Comment:
`can of the` -> `can be the`?

Otherwise, shall we reuse the previous description, `Command that looks like`?